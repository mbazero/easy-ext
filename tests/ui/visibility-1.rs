use easy_ext::ext;

#[ext(StrExt)]
impl str {
    pub const FOO: &'static str = "_";
    const Bar: &'static str = "_"; //~ ERROR All items must have a visibility of `pub`

    pub fn baz(&self) -> String {}
}

fn main() {}
