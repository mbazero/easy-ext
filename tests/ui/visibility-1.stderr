error: All items must have a visibility of `pub`
 --> $DIR/visibility-1.rs:6:11
  |
6 |     const Bar: &'static str = "_"; //~ ERROR All items must have a visibility of `pub`
  |           ^^^
